src/renderers/webgl/WebGLState.js

```diff
@@ -336,6 +336,8 @@ function WebGLState( gl, extensions, paramThreeToGL ) {
 
 	var maxTextures = gl.getParameter( gl.MAX_TEXTURE_IMAGE_UNITS );
 
+	var lineWidthAvailable = parseFloat(gl.getParameter(gl.VERSION).split(' ')[1]) >= 1.0;
```

Comment:
I think you should make this code more robust to maintenance releases. Future WebGL releases might be numbered 2.0.1, for example.